packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -1,6 +1,49 @@
-import { mySqlPasswordClient } from './clients/mysql/MySQLPasswordClient';
+import { SSMClient, GetParameterCommand } from '@aws-sdk/client-ssm';
+// @ts-ignore
+import { DBAdapter, DBConfig, getDBAdapter } from 'rds-query-processor';
 
-export const run = async (event) => {
-  const result = await mySqlPasswordClient.executeRequest(event);
+let adapter: DBAdapter;
+let secretsClient: SSMClient;
+
+export const run = async (event): Promise<any> => {
+  if (!adapter) {
+    const config = await getDBConfig();
+    adapter = await getDBAdapter(config);
+  }
+  const result = await adapter.executeRequest(event);
   return result;
 };
+
+const createSSMClient = (): void => {
+  secretsClient = new SSMClient({});
+}
+
+const getSSMValue = async(key: string | undefined): Promise<string> => {
+  if (!key) {
+    throw Error('Key not provided to retrieve database connection secret');
+  }
+  const parameterCommand = new GetParameterCommand({
+    Name: key,
+    WithDecryption: true,
+  });
+  const data = await secretsClient.send(parameterCommand);
+  if ((data.$metadata?.httpStatusCode && data?.$metadata?.httpStatusCode >= 400) || !data.Parameter?.Value) {
+    throw new Error('Unable to get secret for database connection');
+  }
+  return data.Parameter?.Value ?? '';
```

Comment:
Why an empty string by default? Isn't that an error case as well?

Reply:
I see this logic is carried forward from existing code. We can directly return `data.Parameter?.value`, I will test and add it in a follow up PR.

Reply:
Removed the default empty string and added a statement to throw exception if any of the config doesn't exist.